### PR TITLE
Disable and remove some card views

### PR DESCRIPTION
**Pull request by richard-cox**
- Now the only list that uses both card and table views is the app wall
- Remove associated card components
- cf/org/space filters are now disabled to start with rather than hidden
- Fixed right padding of last div in list header

### CODE REVIEW

**Comment by irfanhabib** [comment]
As part of this, I think the `AppEventDetailDialogComponentComponent` component needs to be removed as well, since its no longer used. Its under `src/app/shared/components/cards/custom-cards/card-app-event`

**Comment by richard-cox** [comment]
Ah, knew I should have added a comment. That will eventually be used as part of https://github.com/SUSE/stratos/issues/170

File: src/app/shared/components/table/table-cell-actions/table-cell-actions.component.ts [hunk 1]
```diff
@@ -1,5 +1,4 @@
 import { CardComponent } from '../../cards/card/card.component';
```
Comment (irfanhabib) [inline src/app/shared/components/table/table-cell-actions/table-cell-actions.component.ts]: component no longer in use

**Comment by irfanhabib** [comment]
LGTM. One unused import needs to be removed.

**Pull request merged**